src/ConfigAtom.hpp

```diff
@@ -356,23 +357,25 @@ namespace configmaps {
       return parsed = true;
     }
 
-    inline bool parseBool() {
-      if(type != UNDEFINED_TYPE) {
-        throw std::runtime_error("ConfigAtom parsing wrong type line ...");
-      }
-      sValue = trim(sValue);
-      if(sValue == "true" || sValue == "True" || sValue == "TRUE") {
-        iValue = 1;
-        return parsed = true;
-      }
+  inline bool parseBool() {
+  if(type != UNDEFINED_TYPE) {
+    throw std::runtime_error("ConfigAtom parsing wrong type line ...");
+  }
+  sValue = trim(sValue);
+  if(sValue == "true" || sValue == "True" || sValue == "TRUE") {
+    iValue = true;
+    return parsed = true;
+  }
+
+  if(sValue == "false" || sValue == "False" || sValue == "FALSE") {
+    iValue = false;
+    return parsed = true;
+  }
+
+  return parsed = false;
+}
 
-      if(sValue == "false" || sValue == "False" || sValue == "FALSE") {
-        iValue = 0;
-        return parsed = true;
-      }
 
-      return parsed = sscanf(sValue.c_str(), "%d", &iValue);
```

Comment:
I think this line allowed to also define the boolean state via an int number (0 / 1). Maybe we could keep that feature?

Reply:
It would also be easier to track the changes if you keep the original indentation.